src/utils/constraints.ts

```
@@ -16,82 +28,381 @@ export interface BathroomItem {
+  type: WallType;
+  position: Position;
+  distance: number;
+}
+
+// Find the nearest wall to a given position
+export const findNearestWall = (
   position: Position,
   roomWidth: number,
   roomHeight: number,
   objectType: ComponentType | null = null,
   scale: number = 1.0
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)